### ISSUE TITLE
Harfbuzz version too old (1.3.1)

### ISSUE THREAD

**Issue by unixall** [ToastWallet/core#53]
ToastWallet 3.0.0 is failing to start on Archlinux x64

> `./ToastWallet-3.0.0-x86_64.AppImage 
> 
> (toastwallet:2926): Pango-ERROR **: 19:08:53.553: Harfbuzz version too old (1.3.1)
> 
> Trace/breakpoint trap (core dumped)`

**Comment by codetsunami** [comment]
Hey

Thanks for the report

Can you try build it on your system and see if the error still occurs then?

Build script is here
https://github.com/ToastWallet/build-scripts/tree/master/linux

If it does and you can figure out the cause please open a pull with a fix!

**Comment by unixall** [comment]
It needs to be rebuild against a newer version of harfbuzz

**Comment by codetsunami** [comment]
Yep looks like an electron build issue. https://github.com/electron/electron/issues/20348

In the meantime the browser version should work for you (through chrome) or try the windows version through wine?

**Comment by unixall** [comment]
Are you using Electron 1.6.2? Perhaps updating to 6.0.1 will fix this?

**Comment by unixall** [comment]
The issue has reportedly been fixed in Electron v4.2.x or later. Which version is Toast built against? @codetsunami

**Comment by unixall** [comment]
Will this be resolved?

**Comment by btong** [comment]
I have this problem on Fedora linux, and I can't use the instructions on the build page because they do not work for Fedora: phonegap and icnsutils are not in the repo.

**Comment by btong** [comment]
Wine gives the error "wine: Bad EXE format" with both the portable and the Windows 7 exes.

**Comment by btong** [comment]
Anything I can do to help?

**Comment by maf2** [comment]
The same problem on Ubuntu 20.04

(toastwallet:65061): Pango-ERROR **: 03:46:40.889: Harfbuzz version too old (1.3.1)

**Comment by Vantskruv** [comment]
I found a solution for this problem, as I have been trying to look up all of my crypto-accounts which I got years ago.
This post helped me (I am using ArchLinux). It applies on another wallet, but it works for ToasWallet aswell.


`
curl -JOL http://ftp.acc.umu.se/pub/GNOME/sources/pango/1.42/pango-1.42.4.tar.xz
tar xf pango-1.42.4.tar.xz
cd pango-1.42.4
printf "all:\n\ttrue\n\ninstall:\n\ttrue\n\n" > tests/Makefile.in
./configure --prefix=/usr
make && make DESTDIR="~/oldlib/pango" install

LD_LIBRARY_PATH=~/oldlib/pango/usr/lib /usr/share/Dagcoin-wallet/DagWallet
`

Src: https://bbs.archlinux.org/viewtopic.php?pid=1866410#p1866410
In the code above, replace
/usr/share/Dagcoin-wallet/DagWallet
with the ToasWallet-xxxxx.appimage you are using.

Luckily I found my ToastWallet folder in ~/.config on a backup drive, and copied it to my current ~/.config folder, before I started ToastWallet. Then I migrated it XUMM, here is a guide.
https://support.xumm.app/hc/en-us/articles/360018135840-How-to-migrate-from-Toast-Wallet-to-XUMM

Edit: Or just save your public adress and the revealed secret to a paper, if you still want to hold XRP.

**Comment by biaskoc** [comment]
I still have XRP on my toastwallet, now I would like to send it to my Xaman wallet.
Unfortunately I don't have the passprhase anymore, if I want to restore the wallet with the source code from the email, that works well, but the reset passprhase doesn't work, I can't bring up the secret either.
I still have the original toastwallet on my phone and I also tried it with the desktop version from github / RichardAH which version still works or am I doing something wrong please help me out
greetings biaskoc
